app/src/main/java/com/gruppo3/wetravel/mapManager/types/ViewMap.java

```diff
@@ -80,8 +84,9 @@ public LocationRequest getLocationRequest() {
     }
 
     /**
-     * Returns the current {@link ViewMap} assigned zoom level.
-     * @return Integer of the default zoom level currently associated with this ViewMap
+     * Returns the current ViewMap assigned zoom level.
+     *
+     * @return Integer of the default zoom level currently associated with this ViewMap.
```

Comment:
You removed `{@link}` in here...
If you made it on purpose, because it looks redundant for you, I respect your choice.
Also, does it always return a default zoom level?
You might consider this suggestion...
```suggestion
     * Returns the zoom level of the current {@link ViewMap}.
     *
     * @return Integer of the zoom level currently assigned at this {@link ViewMap}.
```